sashimi-plot.py

```diff
@@ -278,15 +278,21 @@ def read_gtf(f, c):
         transcripts = OrderedDict()
         chr, start, end = parse_coordinates(c)
         end = end -1
-        with open(f) as openf:
+        with gzip.open(f, "rt") if f.endswith(".gz") else open(f, "r") as openf:
                 for line in openf:
                         if line.startswith("#"):
                                 continue
                         el_chr, _, el, el_start, el_end, _, strand, _, tags = line.strip().split("\t")
                         if el_chr != chr:
                                 continue
+                        if el not in ("transcript", "exon"):
+                                continue
                         d = dict(kv.strip().split(" ") for kv in tags.strip(";").split("; "))
-                        transcript_id = d["transcript_id"]
+                        try:
+                                transcript_id = d["transcript_id"]
+                        except KeyError as e:
+                                print(line)
+                                exit(e)
```

Comment:
```suggestion
                        except KeyError:
                                print("ERROR: 'transcript_id' attribute is missing in GTF file.")
                                exit(1)
```
Maybe better if we do not print the GTF line, but rather provide a more clear error message, following this structure (analogous to the one employed for other error messages along the code).